modules/building-memcached-operator-using-osdk.adoc

```diff
@@ -69,6 +70,60 @@ update the generated code for that resource type:
 $ operator-sdk generate k8s
 ----
 
+. *Optional: Add custom validation to your CRD.*
++
+OpenAPIv3 schemas are added to CRD manifests in the `spec.validation` block when
+the manifests are generated. This validation block allows Kubernetes to validate
+the properties in a Memcached CR when it is created or updated.
```

Comment:
Because you spell out CRD on first instance, you might want to also follow the convention for CR here.

Reply:
It's spelled out in an earlier step of this procedure.